root/etc/s6-overlay/s6-rc.d/init-mod-jellyfin-amd-add-package/run

```diff
@@ -8,21 +8,17 @@ if [ ! -f "/usr/bin/apt" ]; then
     exit 0
 fi
 
-if ! dpkg -l | grep gnupg > /dev/null; then
-    apt-get update && apt-get install -y gnupg
-fi
-
 if [ ! -f "/etc/apt/sources.list.d/kisak-mesa.list" ]; then
     echo "**** Adding kisak-mesa repo ****"
-    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F63F0F2B90935439
+    gpg --homedir /tmp --no-default-keyring --keyring /usr/share/keyrings/kisak-mesa.gpg --keyserver keyserver.ubuntu.com --recv-keys F63F0F2B90935439
```

Comment:
```suggestion
    curl -fsSL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xEB8B81E14DA65431D7504EA8F63F0F2B90935439" | gpg --dearmor | tee /usr/share/keyrings/kisak-mesa.gpg >/dev/null
```
if we want to stylize it

Reply:
Nice find! I like that a lot better since the gpg method leaves a temporary keyring behind